src/util.js

```diff
@@ -116,14 +117,15 @@ exports.deserialize = (data, callback) => {
 /**
  * Get the CID of the DAG-Node.
  *
- * @param {Object} dagNode - Internal representation
+ * @param {Buffer} blob - Serialized binary data
```

Comment:
The Github UI isn't good with changes. So I repost it (slightly different though :) I would say something like `Encoded CBOR blob` or `Serialized as CBOR`.